lib/dispatcher.js

```diff
@@ -151,7 +151,7 @@ Dispatcher.prototype.dispatch = function(params, dispatchOptions) {
         me.authenticator.authenticateRequest(params);
       }
       params.headers = _.extend(
-          {}, me.defaultHeaders, dispatchOptions.headers || {});
+          {}, params.headers, me.defaultHeaders, dispatchOptions.headers || {});
       me._addVersionInfo(params);
```

Comment:
Then we add the version header here, meaning that users of the library *can't* overwrite this header - which I'm totally cool with. Just an interesting thing I noticed here, but I think we can keep this behavior?

Reply:
Also, aside: any idea why Github chooses to place comments on the left vs. right?

Reply:
Yes, I think we should keep it. Also, GitHub puts it on the left when the lines aren't changed.